Chatify/chat/templates/chat/base.html

```diff
@@ -3,30 +3,25 @@
 <html lang="en">
 <head>
         <meta charset="UTF-8">
-        <title>{% block title %}
-    {% endblock %}</title>
-        <link rel="stylesheet" href="{% static 'css/register.css' %}">
-        <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+        <title>{% block title %} {% endblock %}</title>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
         <script src="{% static 'js/jquery.min.js' %}"></script>
-        <script src="{% static 'js/angular.min.js' %}"></script>
-        <script src="{% static 'js/ajax_request.js' %}"></script>
-        <script src="{% static 'js/register.js' %}"></script>
-        <script src="{% static 'js/bootstrap.bundle.min.js' %}"></script>
-        <script src="{% static 'js/login.js' %}"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
+        <script type="text/javascript"
+        src="{% static 'js/jquery.mCustomScrollbar.min.js' %}"></script>
+        <script src="{% static 'js/chat.js' %}"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+
+        <link rel="stylesheet" href="{% static 'css/register.css' %}">
```

Comment:
<link rel="stylesheet" href="{% static 'css/register.css' %}"> should be in register html page